modules/tinymce/src/core/main/ts/mode/Disabled.ts

```diff
@@ -0,0 +1,144 @@
+import { Arr, Optional, Strings } from '@ephox/katamari';
+import { Attribute, Compare, SelectorFilter, SelectorFind, SugarElement } from '@ephox/sugar';
+
+import Editor from '../api/Editor';
+import VK from '../api/util/VK';
+import * as ModeUtils from '../util/ModeUtils';
+
+const isDisabled = (editor: Editor): boolean => editor.disabled;
+
+const internalContentEditableAttr = 'data-mce-contenteditable';
+
+const setContentEditable = (elm: SugarElement<HTMLElement>, state: boolean) => {
+  elm.dom.contentEditable = state ? 'true' : 'false';
```

Comment:
Do we need that condition? We know that `state` is always going to be a boolean so we could just use `toString`:

`
const setContentEditable = (elm: SugarElement<HTMLElement>, state: boolean) => {
  elm.dom.contentEditable = state.toString();
`

Reply:
👍  Changed to use `ContentEditable.set` instead.